core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -193,17 +193,17 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
     }
 
     // pkg private for testing
-    void execute(Terminal terminal, String pluginId, boolean isBatch, Environment env) throws Exception {
-        if (pluginId == null) {
-            throw new UserException(ExitCodes.USAGE, "plugin id is required");
+    void execute(Terminal terminal, String pluginName, boolean isBatch, Environment env) throws Exception {
```

Comment:
Please revert the name change. It's fine as is, and the name change here isn't thorough enough (there are methods invoked here that still have the parameter named `pluginId`).